js/js.tests/test/org/jetbrains/kotlin/generators/tests/GenerateJsTests.kt

```diff
@@ -85,6 +77,36 @@ fun main(args: Array<String>) {
                 model("codegen/boxError", targetBackend = TargetBackend.JS_IR)
             }
 
+            testClass<AbstractIrCodegenBoxWasmTest> {
+                model(
+                    "codegen/box", pattern = "^([^_](.+))\\.kt$", targetBackend = TargetBackend.WASM, excludeDirs = listOf(
+
+                        // JVM-specific
+                        "assert", "builtinStubMethods",
+
+                        // TODO: Support reflection
+                        "toArray", "classLiteral", "reflection",
```

Comment:
Just wondering, why do you just not ignore those tests?

Reply:
Historical reasons. It was less intrusive when most tests were failing.